_posts/en/newsletters/2020-02-12-newsletter.md

```diff
@@ -0,0 +1,111 @@
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #17578][] FIXME:dongcarl rpc: deprecate getaddressinfo label
+
+- [Bitcoin Core #18032][] extends the results of the `createmultisig`
+  and `addmultisigaddress` RPCs to include a `descriptor` field that
+  contains an [output script descriptor][topic output script descriptors]
+  for the created multisig address.  Descriptors make it easy to copy
+  essential information related to the address that will be necessary to
+  create unsigned transactions that spend money received to the address.
+
+- [C-Lightning #3475][] allows a plugin hook to return `{ "result" :
+  "continue" }` to tell `lightningd` to process the action the same way
+  it would without the hook being executed.  This makes it easy for
+  hooks to only execute in special cases.
+
+- [C-Lightning #3372][] allows the user to specify an alternative
+  program to use instead of one of the default sub-daemons (the
+  C-Lightning system consists of multiple interacting daemons, referred
+  to as *sub-daemons* of `lightningd`).  For example (from the PR
+  description):
+
+      # Use remote_hsmd instead of lightning_hsmd for signing:
+      lightningd --alt-subdaemon=lightning_hsmd:remote_hsmd ...
+
+  This option can be dangerous if the alternative sub-daemon isn't fully
+  compatible with the other daemons being used, but it also allows
+  improved flexibility and may simplify some testing.
+
+- [C-Lightning #3465][] implements anti fee sniping for withdrawal
+  transactions, similar to LND's implementation of the same thing (see
+  [Newsletter #18][news18 afs]).  Anti fee sniping uses the nLockTime
+  field to prevent a transaction from being included in any block whose
+  height is lower than that of the tip of the block chain when the
+  transaction was produced.  This limits the ability of a miner who is
+  reorganizing (forking) the chain from being able to arbitrarily
+  rearrange transactions to maximize their fee revenue.
+
+- [LND #3957][] adds some code that can be used in later PRs that add
+  Atomic Multipath Payments (AMP).  AMP is another type of [multipath
+  payment][topic multipath payments] similar to the "base" or "basic"
+  type already supported by C-Lightning, Eclair, and LND.  AMP is harder
+  for routing nodes to distinguish from normal single-part payments and
+  can guarantee that the receiver either claims all parts of the payment
+  or none of it.
+
+- [BOLTs #684][] updates [BOLT7][] to suggest that nodes send their own
+  generated announcements even when the remote peer requests a filter
+  that would suppress that announcement.  This can help ensure that a
+  node gets announced to the network without significantly reducing the
+  overall savings in CPU and bandwidth provided by filtering.
```

Comment:
perhaps: "provided by filtering gossip messages of peers where propagation seems adequate."

Reply:
That makes the sentence very long, and I'm not sure it's exactly what Dave is trying to communicate. I do think this sentence could be clarified a little though - ".. reducing the overall savings ..." is reducing a reduction, which is a slightly confusing thing to be talking about.